release-notes/2016-apr.md

```diff
@@ -0,0 +1,521 @@
+ 
+<!--  - name: migrate database -->
+<!--    # XXX (16-Mar-2016) standalone installs aren't ideal, because they don't give -->
+<!-- diff --git a/tasks/install_publishing.yml b/tasks/install_publishing.yml -->
+<!-- index 429c167..ee9e168 100644 -->
+<!-- --- a/tasks/install_publishing.yml -->
+<!-- +++ b/tasks/install_publishing.yml -->
+<!-- @@ -244,12 +244,12 @@ -->
+<!--    #       looking at the traceback line. -->
+<!--    failed_when: "publishing_initdb_cmd.rc > 0 and 'psycopg2.ProgrammingError: type \"publication_states\" already exists' not in publishing_initdb_cmd.stderr" -->
+ 
+<!-- -- name: initialize db-migrator -->
+<!-- -  when: "'already exists' not in publishing_initdb_cmd.stderr" -->
+<!-- -  command: "{{ root_prefix }}/var/cnx/venvs/publishing/bin/dbmigrator --config {{ root_prefix }}/etc/cnx/publishing/app.ini --context cnx-archive --context cnx-publishing init" -->
+<!-- -  -->
+<!-- -- name: migrate database -->
+<!-- -  command: "{{ root_prefix }}/var/cnx/venvs/publishing/bin/dbmigrator --config {{ root_prefix }}/etc/cnx/publishing/app.ini --context cnx-archive --context cnx-publishing migrate" -->
+ 
+<!--  # +++ -->
+<!--  # Init service -->
+<!--   ``` -->
+<!-- - :white_check_mark: checkpoint :tada: -->
+<!-- - Run db-migrator `/var/cnx/venvs/publishing/bin/dbmigrator --config /etc/cnx/publishing/app.ini --context cnx-publishing --context cnx-archive init --version 0` -->
+<!-- - :camera: snapshot -->
+<!-- - Deployed with `environment/vm2/all/vars.yml`: -->
+
+<!--   ```yml -->
+<!-- source_versions: -->
+<!--   "cnx-archive": e3fa7d038607894c281433f9f3828b7597a3128b -->
+<!--   ``` -->
+<!-- - `ansible-playbook -i enviroment/vm2/inventory --vault-password-file ../vault-password.txt main.yml` -->
+<!-- - :camera: snapshot -->
+<!-- - Run db-migrator `/var/cnx/venvs/publishing/bin/dbmigrator --config /etc/cnx/publishing/app.ini --context cnx-archive migrate --version 20160128110515` -->
+<!-- - :camera: snapshot -->
+
+#### Preparation
+
+- install [db-migrator](https://pypi.python.org/pypi/db-migrator) in the virtual-environment that contains cnx-publishing.
+- `dbmigrator init --version 0`
+
+- release [Products.RhaptosModuleStorage](https://github.com/Rhaptos/Products.RhaptosModuleStorage) @ `HEAD`
+
+#### Update code
+
+- cnx-epub @ `HEAD`
+- [cnx-easybake](https://github.com/Connexions/cnx-easybake) @ `HEAD`
+- cnx-archive @ `e3fa7d038607894c281433f9f3828b7597a3128b`
+- cnx-publishing @ `HEAD`
+
+- Products.RhaptosModuleStorage as `egg`
+
+#### Run migrations
+
+This migration runs to a middle of the road solution,
+where things will continue working while we transition to the next stage
+
+`dbmigrator --config <publishing-config-ini> --context cnx-archive migrate --version 20160101000000`
+`dbmigrator --config <publishing-config-ini> --context cnx-archive migrate --version 20160128110515`
```

Comment:
I think these two lines are displaying as one line, maybe we need an extra newline or just use ```